src/style.css

```diff
@@ -0,0 +1,157 @@
+/* #region reset */
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul {
+  margin: 0;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  display: block;
+}
+
+/* #endregion */
+
+/* #region fonts */
+
+@font-face {
+  font-family: "roboto";
+  src: url("./fonts/roboto-regular-webfont.woff2") format("woff2"),
+    url("./fonts/roboto-regular-webfont.woff") format("woff");
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "roboto";
+  src: url("./fonts/roboto-medium-webfont.woff2") format("woff2"),
+    url("./fonts/roboto-medium-webfont.woff") format("woff");
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "roboto";
+  src: url("./fonts/roboto-bold-webfont.woff2") format("woff2"),
+    url("./fonts/roboto-bold-webfont.woff") format("woff");
+  font-weight: 700;
+  font-style: normal;
+}
+
+/* #endregion */
+
+:root {
+  --primary-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --button-color: #00acdc;
+}
+
+html {
+  font-family: "roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  object-fit: contain;
+  object-position: center;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__name {
+  display: block;
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 150%;
+  color: var(--primary-text-color);
+}
+
+.card__description {
+  display: block;
+  margin-bottom: 16px;
+  font-size: 10px;
+  color: var(--secondary-text-color);
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url("./images/star.svg") center no-repeat;
+}
+
+.stars__star--active {
+  background-image: url("./images/star-active.svg");
+}
+
+.card__reviews-amount {
+  font-size: 10px;
+  color: var(--primary-text-color);
+}
+
+.card__price-block {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 17px;
+  font-size: 12px;
+  color: var(--secondary-text-color);
+}
+
+.card__price-tag {
+  display: block;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 1.13;
+  color: var(--primary-text-color);
+}
+
+.card__button {
+  padding: 9px 0 10px;
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  text-align: center;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  color: #fff;
+  background-color: var(--button-color);
+  transition-property: background-color, border-color, color;
+  transition-duration: 600ms;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  border-color: var(--button-color);
```

Comment:
you can define this border color into `.card__button` styles and don't change it